eng/common/pipelines/templates/steps/publish-blobs.yml

```diff
@@ -4,6 +4,11 @@ parameters:
   TargetLanguage: ''
   BlobName: ''
   ScriptPath: ''
+  RepoId: ''
```

Comment:
This means we will also have to make changes to the release pipelines to start consuming these changes. We need to verify that we don't break if these are not passed in.